examples/all-clusters-app/all-clusters-common/src/microwave-oven-mode.cpp

```diff
@@ -0,0 +1,91 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#include <app-common/zap-generated/attributes/Accessors.h>
+#include <microwave-oven-mode.h>
+
+using namespace chip::app::Clusters;
+using namespace chip::app::Clusters::MicrowaveOvenMode;
+using chip::Protocols::InteractionModel::Status;
+template <typename T>
+using List              = chip::app::DataModel::List<T>;
+using ModeTagStructType = chip::app::Clusters::detail::Structs::ModeTagStruct::Type;
+
+static ExampleMicrowaveOvenModeDelegate gMicrowaveOvenModeDelegate;
+static ModeBase::Instance gMicrowaveOvenModeInstance(&gMicrowaveOvenModeDelegate, 0x1, MicrowaveOvenMode::Id,
```

Comment:
please do not magic number constants. Name 0x01 as kEndpointId.

This also shows that a single endpoint is hardcoded. How do we exemplify multiple endpoint support?